lib/hammer_cli_katello/search_options_creators.rb

```diff
@@ -87,12 +87,15 @@ def create_host_collections_search_options(options, mode = nil)
       search_options
     end
 
-    def create_search_options_with_katello_api(options, resource, _mode = nil)
+    def create_search_options_with_katello_api(options, resource, mode = nil)
       search_options = {}
       searchables(resource).each do |s|
         value = options[HammerCLI.option_accessor_name(s.name.to_s)]
-        if value
+        values = options[HammerCLI.option_accessor_name(s.plural_name.to_s)]
+        if value && [:single, nil].include?(mode)
           search_options.update(s.name.to_s => value.to_s)
+        elsif values && [:multi, nil].include?(mode)
```

Comment:
Don't include nil here. It'll never make it as it's short circuited on L95.

Reply:
What if value == nil && values != nil && mode == nil?

Reply:
Oh, sorry :) I see now this is values, not value.